### PR TITLE
use @embroider/test-setup

### PR DESCRIPTION
**Pull request by ef4**
I have packaged up the best-practices (some of them piloted by this addon, thanks!) for testing addons under embroider in the new library `@embroider/test-setup`. This switches to using it.

### CODE REVIEW

**Review by jelhan — APPROVED** [review]
Great to see that Ember compatibility tests getting easier. I guess you are planning a RFC to add it to the default blueprint soon?

Looks good to me. But like to leave it to @simonihmig to hit the merge button as he has done nearly all Embroider related work.

**Review by simonihmig — APPROVED** [review]
Love it, thank you @ef4!

**Pull request merged**
